File: python/calculate.py
class Calculate():    
    def sum(self,one,two):
        result = one + two
        print(f"this is my result to sum: {result}")

    def rest(self,one,two):
        result = one - two
        print(f"this is my result to rest: {result}")

    def multiplication(self,one,two):
        result = one * two
        print(f"this is my result to multiplication: {result}")

    def division(self,one,two):
        result = one / two
        print(f"this is my result to division: {result}")

x  = Calculate()
while x:
    try:
        one = int(input("enter his firt number: "))
        two = int(input("enter his second number: "))
        option = int(input("enter one option: sum[1], rest[2], multiplication[3], divisions[4]: "))
    except: 
        one = int(input("enter his firt number: "))
        two = int(input("enter his second number: "))
        option = int(input("enter one option: sum[1], rest[2], multiplication[3], divisions[4]: "))

    if option == 1:
        x.sum(one, two)
    elif option == 2:
        x.rest(one, two)
    elif option == 3:
        x.multiplication(one, two)
    elif option == 4:
        x.division(one, two)
    else:
        print("end to program")
        break

File: dart/calculate.dart
import 'dart:io';

class Calculate {
  sum(number_one, number_two){
    var result = number_one + number_two;
    print("this is my result: ${result}");
  }

  rest(number_one, number_two){
    var result = number_one - number_two;
    print("this is my result: ${result}");
  }

  multiplication(number_one, number_two){
    var result = number_one * number_two;
    print("this is my result: ${result}");
  }

  division(number_one, number_two){
    var result = number_one / number_two;
    print("this is my result: ${result}");
  }
}

void main(){
  while (true){
      var x = Calculate();
      print("enter one option: sum[1], rest[2], multiplication[3], divisions[4]: ");
      int option = int.parse(stdin.readLineSync()); 
      print("enter his firt number: ");
      int number_one = int.parse(stdin.readLineSync()); 
      print("enter his second number: ");
      int number_two = int.parse(stdin.readLineSync()); 
      if(option == 1){
        x.sum(number_one, number_two);
      }else if (option == 2){
        x.rest(number_one, number_two);
      }else if (option == 3){
        x.multiplication(number_one, number_two);
      }else if (option == 4){
        x.division(number_one, number_two);
      }else {
        print("option  not found");
        break;
      }
    }
}